### PR TITLE
2.4 upgrade

### PR DESCRIPTION
**Pull request by jwkaltz**

### CODE REVIEW

**Comment by jwkaltz** [comment]
@ger-benjamin @sbrunner I am not sure about the CI changes - do we need to change something more so that the new version will be automagically deployed in our OpenShift?

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -6,13 +6,13 @@ on:
 
 env:
   PROJECT: demo
-  # Requires CI_GPG_PRIVATE_KEY and GITHUB_GOPASS_CI_TOKEN secrets.
-  OPENSHIFT_PROJECT: gs-gmf-demo
-  HELM_RELEASE_NAMES: '2.4' # List of branch that should be deployed on helm
+  # Requires CI_GPG_PRIVATE_KEY and GOPASS_CI_GITHUB_TOKEN secrets.
+  # OPENSHIFT_PROJECT: gs-gmf-demo
+  # HELM_RELEASE_NAMES: # List of branch that should be deployed on helm
```
Comment (sbrunner) [inline .github/workflows/ci.yaml]:
```suggestion
  OPENSHIFT_PROJECT: gs-gmf-demo
  HELM_RELEASE_NAMES: '2.4' # List of branch that should be deployed on helm
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -6,13 +6,13 @@ on:
...
   PATH: /bin:/usr/bin:/usr/local/bin:/home/runner/.local/bin
   SUMMON_PROVIDER: /usr/local/bin/gopass
   OPENSHIFT_URL: https://api.openshift-ch-1.camptocamp.com/
-  OPENSHIFT_RELEASE: prod-2-4
+  # OPENSHIFT_RELEASE: # Optional, name of the OpenShift release
```
Comment (sbrunner) [inline .github/workflows/ci.yaml]:
```suggestion
  OPENSHIFT_RELEASE: prod-2-4 # Optional, name of the OpenShift release
```

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -29,8 +29,10 @@ jobs:
       - uses: camptocamp/initialise-gopass-summon-action@v1
         with:
           ci-gpg-private-key: ${{secrets.CI_GPG_PRIVATE_KEY}}
-          github-gopass-ci-token: ${{secrets.GITHUB_GOPASS_CI_TOKEN}}
+          github-gopass-ci-token: ${{secrets.GOPASS_CI_GITHUB_TOKEN}}
         if: "env.HELM_RELEASE_NAMES != ''"
+      # - run: scripts/deploy-docker --image=config --no-trigger
+      #   if: "env.HELM_RELEASE_NAMES != ''"
```
Comment (sbrunner) [inline .github/workflows/ci.yaml]:
```suggestion
      - run: scripts/deploy-docker --image=config --no-trigger
        if: "env.HELM_RELEASE_NAMES != ''"
```

File: .github/workflows/ci.yaml [hunk 4]
```diff
@@ -51,7 +53,9 @@ jobs:
       - uses: camptocamp/initialise-gopass-summon-action@v1
         with:
           ci-gpg-private-key: ${{secrets.CI_GPG_PRIVATE_KEY}}
-          github-gopass-ci-token: ${{secrets.GITHUB_GOPASS_CI_TOKEN}}
+          github-gopass-ci-token: ${{secrets.GOPASS_CI_GITHUB_TOKEN}}
         if: "env.HELM_RELEASE_NAMES != ''"
+      # - run: scripts/deploy-docker --image=geoportal --no-trigger
+      #   if: "env.HELM_RELEASE_NAMES != ''"
```
Comment (sbrunner) [inline .github/workflows/ci.yaml]:
```suggestion
      - run: scripts/deploy-docker --image=geoportal --no-trigger
        if: "env.HELM_RELEASE_NAMES != ''"
```

File: CONST_Makefile [hunk 5]
```diff
@@ -291,22 +291,22 @@ CONFIG_VARS += sqlalchemy.url sqlalchemy.pool_recycle sqlalchemy.pool_size sqlal
...
 	reset_password fulltextsearch global_headers headers authorized_referers hooks stats db_chooser \
 	ogcproxy_enable dbsessions urllogin host_forward_host smtp c2c.base_path welcome_email \
-	lingua_extractor interfaces_config interfaces default_interface devserver_url api
+	lingua_extractor interfaces_config interfaces default_interface devserver_url api pdfreport
 
 # Secrets
 
 ifndef DISABLE_DEFAULT_GPG_KEYS
 GPG_KEYS += 5E026E27C9F78F65223CB546B39FC793D1D6A94C # Stéphane Brunner
+GPG_KEYS += 3A99C902C2A580A676F54F727C661185B42AF223 # Patrick Valsecchi
```
Comment (sbrunner) [inline CONST_Makefile]:
```suggestion
```

File: CONST_Makefile [hunk 6]
```diff
@@ -291,22 +291,22 @@ CONFIG_VARS += sqlalchemy.url sqlalchemy.pool_recycle sqlalchemy.pool_size sqlal
...
 GPG_KEYS += 1C3E8BAD07E68AF2E15D7C2A3C193EAE3EA11D26 # Elisabeth Leu
 GPG_KEYS += 933FFD84E4EB0E12B7B5D0C8EDCE1D2A0875810F # Frederic Junod
 GPG_KEYS += 14B0A8BBEF59FB4984E1A91D0F85749A1DAFEE6A # Oliver Christen
 GPG_KEYS += 7F816541422C98E9A21CC2C3E9461427C8A0FF72 # Wolfgang Kaltz
+GPG_KEYS += 4E98BB3AD36285EFDA47B83B4662C60FA510717D # Michael Künzli
 GPG_KEYS += 1AC7B924F73B4C4A302F39E6EA57F57AA68C2F5F # Arnaud Morvan
 GPG_KEYS += 0545262E5AE04436C83B654D4CEC60DAC4F68132 # Andrea Borghi
 GPG_KEYS += B3771480D024F905F6FA4637BE875E27AAA3DDF0 # Renata Mueller
-GPG_KEYS += C1599283494601AA15A502C9002B1BA74D3386D2 # Benjamin Gerber
-GPG_KEYS += C1D48DC23F6505C929C86C1CA16CD36473EF733C # Cécile Vuilleumier
```
Comment (sbrunner) [inline CONST_Makefile]: Required

Comment (jwkaltz) [reply CONST_Makefile]: I put them back, now

**Review by sbrunner — APPROVED** [review]
Looks good, thanks :-)

**Pull request merged**